main.go

```diff
@@ -108,6 +117,36 @@ func (p *CfPlugin) GetNewAppRoutes(appName string, defaultCfDomain string, manif
 	return uniqueRoutes
 }
 
+func (p *CfPlugin) GetScaleFromManifest(appName string, defaultCfDomain string,
+	manifestReader manifest.ManifestReader) AppScaleParameters {
+	manifest := manifestReader.Read()
+	scaleParams := AppScaleParameters{}
+	appParams := manifest.GetAppParams(appName, defaultCfDomain)
+	if appParams != nil {
+		scaleParams = AppScaleParameters{
+			Memory:        appParams.Memory,
+			InstanceCount: appParams.InstanceCount,
+			DiskQuota:     appParams.DiskQuota,
+		}
+	}
+	return scaleParams
+}
+
+func (p *CfPlugin) MergeScaleParameters(liveAppScale, manifestAppScale AppScaleParameters) (
+	scaleParameters AppScaleParameters) {
```

Comment:
Yeah. We strugged to keep the line within 100 chars width (which is good for general readability/browseability of code)  and this is what we came up with. Putting the linebreak between the `) (` is acually an error. AppScaleParameters will be renamed to ScaleParams though, or possibly replaced by GetAppModel so line becomes shorter.